test/e2e/basic/backup-pvc-config.go

```diff
@@ -0,0 +1,105 @@
+package basic
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/vmware-tanzu/velero/test"
+	. "github.com/vmware-tanzu/velero/test/e2e/test"
+	. "github.com/vmware-tanzu/velero/test/util/k8s"
+	. "github.com/vmware-tanzu/velero/test/util/velero"
+)
+
+type BackupPVCConfigChange struct {
+	TestCase
+	data                      map[string]string
+	configmapName             string
+	namespace                 string
+	sourcePVCStorageClassName string
+	backupPVCStorageClassName string
+	pvcName                   string
+	volumeName                string
+	podName                   string
+	deploymentName            string
+}
+
+var BackupPVCConfigChangeTest = TestFunc(&BackupPVCConfigChange{})
```

Comment:
It seems it should be 
`var BackupPVCConfigChangeTest func() = TestFunc(&BackupPVCConfigChange{})`

Reply:
I think thats fine, type can be omitted there. Framework errors got resolved without changing this.